content/blog/scheduling-in-react/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Scheduling in React"
+date: "2019-03-15T12:00:00.000Z"
+---
+
+In modern applications, user interfaces often have to juggle multiple tasks at the same time: A search component needs to respond to user input while providing auto completion results; An interactive dashboard needs to update the charts while loading data from the server and sending analytics data to your backend.
+
+All these parallel steps can lead to janky interfaces and unhappy users, so let’s learn how we can fix this.
```

Comment:
I use janky ALL THE TIME but I don't know that everyone would know what it means / it feels colloquial and super informal to me. Could we use another word? Like unreliable maybe? What do you think?

"All these parallel steps can lead to janky interfaces and unhappy users"